Userland/Applications/Maps/main.cpp

```diff
@@ -25,6 +26,7 @@ ErrorOr<int> serenity_main(Main::Arguments arguments)
 
     TRY(Core::System::unveil("/res", "r"));
     TRY(Core::System::unveil("/tmp/session/%sid/portal/config", "rw"));
+    TRY(Core::System::unveil("/tmp/session/%sid/portal/launch", "rw"));
```

Comment:
You can remove this line it is already committed in: https://github.com/SerenityOS/serenity/commit/264782557ead5b6e655f1c40d6ad019dc52f660b:
- Remove this line and git commit --ammend
- Sync your master in https://github.com/cubiclove/serenity
- git checkout master
- git pull
- Checkout this branch
- git rebase master
- git push -f